pkg/volume/util/util.go

```diff
@@ -582,3 +583,18 @@ func HasMountRefs(mountPath string, mountRefs []string) bool {
 	}
 	return false
 }
+
+//WriteVolumeCache flush disk data given the spcified mount path
+func WriteVolumeCache(deviceMountPath string, exec mount.Exec) error {
+	// If runtime os is windows, execute Write-VolumeCache powershell command on the disk
+	if runntime.GOOS == "windows" {
```

Comment:
This can stay `runtime` given `apiruntime "k8s.io/apimachinery/pkg/runtime"` above?

Reply:
yes, my mistake. fixed.